packages/backend-common/src/service/ServiceBuilderImpl.ts

```diff
@@ -82,9 +89,20 @@ export class ServiceBuilderImpl implements ServiceBuilder {
         logger.error(`Failed to start up on port ${port}, ${e}`);
         reject(e);
       });
-      const server = app.listen(port, () => {
-        logger.info(`Listening on port ${port}`);
+      const server = stoppable(
+        app.listen(port, () => {
+          logger.info(`Listening on port ${port}`);
+        }),
+        0,
+      );
+
+      useHotEffect(this.module, () => {
+        return () =>
```

Comment:
Maybe I should remove outer function, kinda made sense for the other use case, but here it looks ridiculously :)

Reply:
Also, maybe then `useHotCleanup`

Reply:
Agreed 👍